Create CONTRIBUTING and onboarding document (#68)
alan-turing-institute/asg-community: onboarding.md

The vision of this project is to _Demonstrate, showcase and amplify the real-world impact of ASG research._

ASG researchers will collaborate, exchange knowledge and showcase their work through their involvement in the ASG community and white paper project.
The goal is to collaboratively develop a series of white papers as well as supporting communication outputs highlighting ASG research.
In this document, we provide list of information and resources available for all contributors to the ASG white papers.

## Get in touch with the project team

As a community-led project, there are several contributors to this project (see: https://github.com/alan-turing-institute/asg-community#who-is-involved).
For onboarding and other questions related to this project, you can directly reach out to the Community Manager by emailing `arao@turing.ac.uk`, or contact via Slack.
You can also create [a GitHub issue](https://github.com/alan-turing-institute/asg-community/issues/new) if you would like to discuss your idea openly on this repository.

## Long list of projects

The ASG Community Manager has compiled a long list of a project curated by ASG project leads and Research Application Managers. 

> Please browse through the information provided in this document (READ ONLY): [Link to be added by Achintya].

Reviewer: ```suggestion
Please browse through the information provided [in this document](https://thealanturininstitute-my.sharepoint.com/:x:/g/personal/arao_turing_ac_uk/ETV-miIocs1Lv4Ss305GLG8B45egy4bx-NyP480dHBe3JQ?e=c6NbHU) (internal to the Turing).
```
